subsystems/motor/speedmodifiers/CapSpeedModifier.java

```diff
@@ -0,0 +1,23 @@
+package org.usfirst.frc4904.standard.subsystems.motor.speedmodifiers;
+
+
+public class CapSpeedModifier implements SpeedModifier {
+	public final double maxSpeed;
+	public final double minSpeed;
+	
+	public CapSpeedModifier(double max, double min) {
+		maxSpeed = max;
```

Comment:
By convention we use the same name for constructor arguments that we do for properties. So this line would look like
```
this.maxSpeed = maxSpeed
```